### PR TITLE
feat: add tracking

### PR DESCRIPTION
**Pull request by varholak-peter**
https://jira.economist.com/browse/WP-208

### CODE REVIEW

**Comment by yasgreen93** [comment]
@varholak-peter What ticket is this on jira? Can you put it in the description please? 😊

**Comment by yasgreen93** [comment]
LGTM :rainbow_flag:

**Comment by tomiberes** [comment]
LGTM

**Comment by varholak-peter** [comment]
What is with this ticket? It needs to be merged before https://github.com/EconomistDigitalSolutions/fe-blogs/pull/650

File: src/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* global window document */
+ /* global window document */
```
Comment (umbobabo) [inline src/index.js]: No useless empty spaces pls.

**Pull request merged**